src/middleware/custom.rs

```diff
@@ -9,6 +8,7 @@ use plonky2::field::types::Field;
 use crate::{
     middleware::HASH_SIZE,
     middleware::{hash_fields, Hash, Key, NativePredicate, Params, ToFields, Value, F},
+    Error,
```

Comment:
Can we import `Result` here instead?

Reply:
Agree, done at https://github.com/0xPARC/pod2/compare/d9453797e2f437aefa43712667dc2cc9aac3e20a..92fcdf35427c4f2f90264530b0a27032e2adc170